src/utils/format.py

```diff
@@ -51,11 +51,14 @@ def format_amount(amount, decimals=18, rounding=ROUND_DOWN):
 
   return f'{rounded_value:,.{decimals}f}'.rstrip('0').rstrip('.')
 
+
 def format_date(date):
-  return '' if date is None else date.strftime("%d/%m/%y")
+  return '' if date is None else date.strftime(DATE_FORMAT)
+
+
+def format_date_time(date, tooBigLabel='Never'):
+    return tooBigLabel if date == datetime.max else date.strftime(DATE_TIME_FORMAT_ISO_8601)
```

Comment:
I wouldn't modify the default formatter, so we can leave `format_date_time` as it was, for normal formatting, and create an additional `format_date_time_iso`

Also, I think python3 allows to do this:
`my_date.isoformat()`

I like that you declared constants for the format :)